db/query.go

```diff
@@ -368,7 +374,8 @@ func (context *DatabaseContext) buildChannelsQuery(channelName string, startSeq
 		index = sgIndexes[IndexAllDocs]
 	}
 
-	channelQueryStatement := replaceSyncTokensQuery(channelQuery.statement, context.UseXattrs())
+	channelQueryStatement := replaceActiveOnlyFilter(channelQuery.statement, activeOnly)
```

Comment:
QueryStarChannel doesn't include $$activeOnlyFilter.    It seems like it should be modified to work in the same way, given that it's already returning flags.

Reply:
Added changes to include $$activeOnlyFilter in star channel query.